Update planning usage (#1414)
open-contracting/standard: docs/guidance/map/unsuccessful_processes.md

In most jurisdictions, if a procedure is cancelled or unsuccessful, and a **new procedure** is started to procure the same items, the two procedures are considered two **different** contracting processes. This is in keeping with the OCDS definition of a contracting process.

> But in other jurisdictions, such as Paraguay, the planning is considered as the initiation of the process. In these jurisdictions when a tender fails and a new tender is started, the two tenders are considered part of the same contracting process. This differs from the OCDS definition of a contracting process.

Reviewer: ```suggestion
But in other jurisdictions, such as Paraguay, planning is considered as initiating the process. In these jurisdictions when a tender fails and a new tender is started, the two tenders are considered part of the same contracting process. This differs from the OCDS definition of a contracting process.
```

> In OCDS, it is relevant and desirable to include the planning information that relates to the process, but the contracting process is not interpreted as ‘starting’ with the planning. In OCDS, the planning process is something that comes **before** the initiation of a contracting process. The initiation of the procedure is not the planning process, because at least one of the following is true of a planning process: it is not a concrete attempt to award one or more contracts like a request for tender, etc.; it is not a concrete opportunity for potential suppliers to participate in; it does not describe the competitive conditions.

Reviewer: ```suggestion
In OCDS, it is relevant and desirable to disclose information about planning, but the contracting process is not interpreted as "starting" with planning. In OCDS, the planning process is something that comes **before** the initiation of a contracting process. The initiation of the procedure is not the planning process, because at least one of the following is true of a planning process: it is not a concrete attempt to award one or more contracts like a request for tender, etc.; it is not a concrete opportunity for potential suppliers to participate in; it does not describe the competitive conditions.
```

The [Sistema de Información de las Contrataciones Públicas (SICP)](https://contrataciones.gov.py/) discloses information about contracting processes for all public entities in Paraguay. SICP is managed by the National Directorate of Public Procurement (DNCP in Spanish).

The first data disclosed is about the planning process. Planning data includes an estimate of what an entity is going to buy, when and for how much. SICP assigns an `ocid` when the planning data is first disclosed, before the tender stage. In this example, the ocid is 'ocds-03ad3f-331547'.

```{jsoninclude} ../../examples/unsuccessful-tender-planning.json
:jsonpointer:
:expand: releases, planning
:title: unsuccessful-tender-planning
```

> Next, the tender process is disclosed, using a new `ocid`, 'ocds-03ad3f-331547-1'. The `relatedProcess` block links the planning process and the tender process, with the relationship set to 'planning'.

Reviewer: ```suggestion
Next, the contracting process is disclosed, using a new `ocid`, 'ocds-03ad3f-331547-1'. The `relatedProcess` block links the planning process and the contracting process, with the relationship set to 'planning'.
```

The tender was unsuccessful, so the tender status is set to ‘unsuccessful’.

```{jsoninclude} ../../examples/unsuccessful-tender-tender.json
:jsonpointer:
:expand: releases, relatedProcesses, tender, status
:title: unsuccessful-tender-tender
```

> The buyer issues another tender to buy the same item, based on the same planning and the first tender.

Reviewer: ```suggestion
The buyer issues another tender to buy the same item, following from the same planning process.
```

Paraguay considers the two tenders part of the same contracting process. But, in OCDS the two tenders are separate contracting processes.

To construct an `ocid` for the second contracting process, Paraguay adds a consecutive number to the `ocid` of the first process. In this example the new `ocid` is 'ocds-03ad3f-331547-2'

Paraguay could also have used the identifier for the second tender as the `ocid` for the second contracting process.

> The `relatedProcess` block links the two tender processes, with the relationship set to ‘unsuccessfulProcess’, and the initial planning process, with the relationship set to ‘planning’ 

Reviewer: ```suggestion
The `relatedProcess` block links to the unsuccessful contracting process with the relationship set to ‘unsuccessfulProcess’, and to the initial planning process with the relationship set to ‘planning’.
```
